Source/WebGPU/WebGPU/Device.mm

```diff
@@ -189,6 +198,24 @@ static void captureFrame(id<MTLDevice> captureObject)
     m_isLost = true;
 }
 
+static void setOwnerWithIdentity(id<MTLResourceSPI> resource, auto webProcessID)
+{
+    ASSERT([resource respondsToSelector:@selector(setOwnerWithIdentity:)]);
```

Comment:
The delegate is optional hence the `respondsToSelector` check. But from what I can tell all resources we create implement it